andela/activo-api
# #158818060 Endpoint to search for assets

#### What does this PR do?
search for assets

#### Description of Task to be completed?
* create an endpoint to search for assets
* write test for searching assets

#### How should this be manually tested?
* pull from this branch ```ft-endpoint-to-search-for-assets-158818060``` into a new branch
* run ``` pip install -r requrements.txt ```
* endpoint should look like this: ```/assets/search?start=<date>&end=<date>&warranty_start=<date>&warranty_end=<date>```
* run ```pytest ``` to run the test

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158818060 An endpoint to search for assets using date range and warranty

#### Postman Collection link
https://www.getpostman.com/collections/ef4053ac643efd4840a6

#### Screenshots (if appropriate)
<img width="1135" alt="screen shot 2018-07-11 at 7 53 31 am" src="https://user-images.githubusercontent.com/24475008/42555273-975960ba-84df-11e8-94d1-936ae7f37e5f.png">

<img width="1440" alt="screen shot 2018-07-11 at 7 52 44 am" src="https://user-images.githubusercontent.com/24475008/42555282-9d2a8fb4-84df-11e8-9523-d8491ffc706e.png">
#### Questions:
N/A
